sopel/tools/jobs.py

```diff
@@ -139,19 +142,104 @@ class Job(object):
     execute. Current time is used to decide when the job should be executed
     next so it should only be called right after the function was called.
     """
+    @classmethod
+    def kwargs_from_callable(cls, settings, handler):
```

Comment:
I'm not going to attach a note to every single one, but so many new methods here are lacking docstrings. :/

Reply:
Yup, next commit will be full of them. I didn't want to add docstrings at first, because it's all internal. But heh. Eventually, I prefer *with* docstrings!